### DISCUSSION TITLE
Will the author provide a recape of the Yocto project

### DISCUSSION THREAD

**Discussion by neusoft-zhangboqi** [helix-editor/helix#12719] [General]
I want to integrate helix into the Yocto project(poky linux), but there is no available recipe

**Comment by dtjones190** [comment]
See the .bb and .inc recipe in here, tested on scarthgap.
[helix.zip](https://github.com/user-attachments/files/18578419/helix.zip)

Note that this does not package grammars, that looks to be a little bit more of a project. Probably need to add all of the languages as SRC_URIs and then patch languages.toml to point to the local copies.

**Comment by pascalkuthe** [comment]
We do not package helix for distors ourself. Somebody involved in that project would have to packe it
